sql/core/src/test/scala/org/apache/spark/sql/XPathFunctionsSuite.scala

```diff
@@ -28,6 +29,7 @@ class XPathFunctionsSuite extends QueryTest with SharedSparkSession {
   test("xpath_boolean") {
     val df = Seq("<a><b>b</b></a>").toDF("xml")
     checkAnswer(df.selectExpr("xpath_boolean(xml, 'a/b')"), Row(true))
+    checkAnswer(df.select(xpath_boolean(col("xml"), lit("a/b"))), Row(true))
```

Comment:
Could you update them with
`checkAnswer(df.select(xpath_boolean(col("xml"), lit("a/b"))), df.selectExpr("xpath_boolean(xml, 'a/b')"))`

Reply:
Is it really necessary to do so in such an easily observable situation?
@zhengruifeng

Reply:
I think it is fine in this case, since it implicitly compares with above `df.selectExpr("xpath_boolean(xml, 'a/b')")`

Reply:
Got it.